kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -85,3 +85,33 @@ $ scaleCluster.sh -d domain1 -n weblogic-domain-1 -c cluster-1 -r 3
 domain.weblogic.oracle/domain1 patched
 [2021-02-26T19:04:14.466 UTC][INFO] Successfully patched replicas for cluster 'cluster-1'!
 ```
+
+### Script to view the status of any WebLogic cluster
```

Comment:
of any WebLogic cluster -> of a WebLogic cluster (it's understood that it could be any cluster)